src/main/java/com/teksystems/TekcampExerciseSpring/Shared/Dto/UserDto.java

```diff
@@ -0,0 +1,79 @@
+package com.teksystems.TekcampExerciseSpring.Shared.Dto;
+
+
+public class UserDto {
+	
+	 private Long id;
+	 private String userId;
+		private String firstName;
+		private String lastName;
+	    private String email;
+	    private String password;
+	    private String passwordConfirmation;
+	    private boolean duplicate;
+	    public UserDto() {
```

Comment:
nitpicky maybe, but you need to keep your indentation nice and formatted. It's easier for people to read and understand your blocks if they're indented correctly